python/ml4ir/applications/ranking/tests/test_ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import os
+import warnings
+from ml4ir.base.data import ranklib_to_ml4ir
+import pandas as pd
+
+
+warnings.filterwarnings("ignore")
+
+INPUT_FILE = "ml4ir/applications/ranking/tests/data/sample.txt"
+OUTPUT_FILE = "ml4ir/applications/ranking/tests/data/sample_ml4ir.csv"
+QUERY_ID_NAME = 'qid'
+RELEVANCE_NAME = 'relevance'
+KEEP_ADDITIONAL_INFO = 1
+GL_2_CLICKS = 1
+
+class Test_ranklib_to_ml4ir(unittest.TestCase):
```

Comment:
[Minor] @mohazahran People generally use CapWords convention for class names -> https://www.python.org/dev/peps/pep-0008/#id41